start.markdown

```diff
@@ -3,7 +3,64 @@ title: Getting Started
 layout: main
 ---
 
-- [Getting O2](/git)
-- [Building O2](/building)
-- [Running O2](/running)
-- [Further material](/links)
+Get O²
+======
+
+The ALICE O² Software is hosted on [GitHub](https://github.com) under the
+[AliceO2Group](https://github.com/AliceO2Group/) organization. The core package is
+[AliceO2](https://github.com/AliceO2Group/AliceO2).
+
+
+Build O²
+========
+
+In order to build the ALICE O² software we use [aliBuild](https://alisw.github.io/alibuild). Check
+out our [build instructions](https://alice-doc.github.io/alice-analysis-tutorial/building/).
```

Comment:
I would prefer a page tailored to O2 rather than pointing to the analysis tutorial, probably confusing

Reply:
Hi, it's not. We want people to have one guide for Run2 and Run3, that's deliberate.